### PR TITLE
Dev lobby script

### PR DESCRIPTION
**Pull request by Rivalson11**
[15.md](https://github.com/witseie-elen4010/2022-group-lab-011/files/8780332/15.md)

### CODE REVIEW

File: routes/lobby.js [hunk 1]
```diff
@@ -1,13 +1,79 @@
...
+                        // insert user into lobby
+                        .input('account_id', accountId)
+                        .input('player_role', playerRole)
+                        .query('SELECT * FROM dbo.game_lobby;')
+                  })
+                  .then(result => {
+                    console.log("selected")
+                    console.log(result)
+                    if (result.recordset.length === 2) {
+                        console.log("Create game")
```
Comment (robinjonker) [inline routes/lobby.js:55]: To confirm this is to see the actual database implementation, still going to add functionality right?

**Review by robinjonker — CHANGES_REQUESTED** [review]

File: routes/lobby.js (hunk 1 above)
Comment (Rivalson11) [reply routes/lobby.js:55]: correct these unique identifiers will be passed through to the multiplayer game.

Comment (robinjonker) [reply routes/lobby.js:55]: That makes sense, well done on this PR
[code_review_7.md](https://github.com/witseie-elen4010/2022-group-lab-011/files/8780410/code_review_7.md)
e

**Review by robinjonker — APPROVED** [review]
[code_review_7.md](https://github.com/witseie-elen4010/2022-group-lab-011/files/8780418/code_review_7.md)

**Pull request merged**